website/content/userguide/minio.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "setting up MinIO on gokrazy"
+weight: 50
+---
+
+# Setting up MinIO on gokrazy
+
+Since MinIO is a storage solution, one needs to have the permanent storage with
+a filesystem that the linux kernel understands configured. Further instructions
+on how to do this can be found in the [Quickstart](/quickstart/) article or the
+`gokr-packer` output.
+
+This article also assumes that you have an instance directory set up.
+(also see [Quickstart](/quickstart/))
+
+## Configuring the ENV Vars and flags
```

Comment:
Let’s spell out “ENV Vars” as “environment variables” for consistency with https://gokrazy.org/userguide/package-config/